CMakeLists.txt

```diff
@@ -184,6 +186,25 @@ foreach(backend ${built_backends})
     target_compile_definitions(${backend}
       PRIVATE AF_WITH_LOGGING)
   endif()
+
+  if(AF_WITH_STACKTRACE)
+    if(UNIX)
+      find_library(Backtrace_LIBRARY backtrace
+        DOC "libbacktrace.so file for more informative stacktraces. https://github.com/ianlancetaylor/libbacktrace")
+      if(Backtrace_LIBRARY)
+        target_link_libraries(${backend}
+          PRIVATE ${Backtrace_LIBRARY})
+        target_compile_definitions(${backend}
+          PRIVATE
+            BOOST_STACKTRACE_USE_BACKTRACE)
```

Comment:
Hum the different options:
https://www.boost.org/doc/libs/develop/doc/html/stacktrace/configuration_and_build.html
In order not to conflict with the user bqcktrqce, I would prefer AF to use the header only boost backtrace : would that be possible ?

Reply:
I have updated the PR so that it defaults to Basic. You can still enable libbacktrace. also this library can be installed as a static library.